library/std/src/sys/sgx/abi/usercalls/alloc.rs

```diff
@@ -316,9 +316,9 @@ where
 //   | small1 | Chunk smaller than 8 bytes
 //   +--------+
 fn region_as_aligned_chunks(ptr: *const u8, len: usize) -> (usize, usize, usize) {
```

Comment:
pondering (not this PR): this looks an awful lot like [`.align_to(8)`](https://doc.rust-lang.org/nightly/std/primitive.slice.html#method.align_to).